api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java

```diff
@@ -242,10 +242,10 @@ private WorkspaceResponse getWorkspaceImpl(DbWorkspace dbWorkspace) {
     return workspaceResponse;
   }
 
-  private Map<String, FirecloudWorkspaceResponse> getFirecloudWorkspaces(List<String> fields) {
+  private Map<String, FirecloudWorkspaceResponse> getFirecloudWorkspaces() {
     // fields must include at least "workspace.workspaceId", otherwise
     // the map creation will fail
-    return fireCloudService.getWorkspaces(fields).stream()
```

Comment:
👍 to not doing this; it's pretty opaque and likely doesn't  save  much. I really like objects in Java  to be densely populated.

Reply:
Sorry to break to it to you, but it's still sparse after this change. We're just accepting the underlying service's default partial field settings, and making it standard across different entry points. We can debate the best server side representation of the partial workspace, but it is necessary to request partial data from Firecloud for performance reasons.